driver-mapping/src/test/java/com/datastax/driver/mapping/MapperInvalidAnnotationsTest.java

```diff
@@ -261,27 +265,4 @@ public void setNotReadable(int i) {
 
     }
 
-    @Test(groups = "unit", expectedExceptions = IllegalArgumentException.class,
```

Comment:
Why are these tests no longer applicable?  Shouldn't they still be invalid?  Also noticed that the `Invalid14` class still exists (but is no longer used)